myndla-api/src/main/resources/no/ndla/myndlaapi/db/migration/V13__post_upvote_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE post_upvote (
```

Comment:
Liker å legge på dette for å være sikker på at migreringer kan kjøres omigjen uten å feile.
```suggestion
CREATE TABLE IF NOT EXISTS post_upvote (
```